web/tests/test_forms.py

```
@@ -564,3 +571,258 @@ def test_invalid_profile_form(self):
         }
         form = ProfileUpdateForm(data=form_data, instance=self.user)
         self.assertFalse(form.is_valid())
+
+
+class TeachFormTests(TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.8.2)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)